packages/core/src/events/EventHandlers.ts

```diff
@@ -86,29 +95,32 @@ export class EventHandlers extends Handlers<
           return () => el.setAttribute('draggable', 'false');
         },
         events: [
-          defineEventListener('dragstart', (e: DragEvent, id: NodeId) => {
-            e.stopPropagation();
-            e.stopImmediatePropagation();
-
-            // Ensure the Node is selected when it is being dragged
-            this.store.actions.setNodeEvent('selected', id);
-
-            this.store.actions.setNodeEvent('dragged', id);
-
-            EventHandlers.draggedElementShadow = createShadow(e);
-            EventHandlers.draggedElement = id;
-          }),
-          defineEventListener('dragend', (e: DragEvent) => {
-            e.stopPropagation();
-
-            const onDropElement = (draggedElement, placement) =>
-              this.store.actions.move(
-                draggedElement as NodeId,
-                placement.parent.id,
-                placement.index + (placement.where === 'after' ? 1 : 0)
-              );
-            this.dropElement(onDropElement);
-          }),
+          defineEventListener(
+            'dragstart',
+            (e: DragEvent, id: NodeId) => {
+              // Ensure the Node is selected when it is being dragged
+              this.store.actions.setNodeEvent('selected', id);
```

Comment:
We can remove this if we are swithing back to `mousedown` for the select handler